src/content/getting-started/quickstart/k3s/_index.md

```diff
@@ -60,12 +61,13 @@ curl -sfL https://get.k3s.io | INSTALL_K3S_EXEC="--cluster-cidr $POD_CIDR --serv
 ```
 
 The kubeconfig file needs to be modified to set the Kubernetes API endpoint to the public IP of node-b,
-which is obtained from the first field yielded by `hostname -I`.
+which is obtained from the first field yielded by `hostname -I`, and to give the context a name other
+than “default”.
 
 ```bash
 cp /etc/rancher/k3s/k3s.yaml kubeconfig.cluster-b
 IP=$(hostname -I | awk '{print $1}')
-sed -i 's/127.0.0.1/'$IP'/' kubeconfig.cluster-b
+sed -i "s/127.0.0.1/${IP}/;11,12s/default/cluster-b/" kubeconfig.cluster-b
```

Comment:
As discussed in slack, 11,12s/ means replacing between lines 11 and 12, but that's too specific and the kubeconfig could potentially change (I have a k3s kubeconfig here with the context name in line 15)

Reply:
I changed this to use `yq` instead.